### DISCUSSION TITLE
Correct usage of revenue accounts. Negative values?

### DISCUSSION THREAD

**Discussion by thehijacker** [firefly-iii/firefly-iii#9035] [Firefly III] [answered]
I have setup revenu account which is from my eployer. When I receive my paycheck I create new translaction and select that revenue account as source and my bank account as destination. So the value in revenue account is always negative.

Not sure I am using it correctly. I suppose it needs to be positive value?

Thank you.

**Comment by JC5** [comment, marked as answer]
It's correct that it's negative. After all, to give you money the revenue account "loses" it.
